ci-operator/config/openshift/verification-tests/openshift-verification-tests-master__installation-nightly-4.14.yaml

```diff
@@ -174,8 +174,8 @@ tests:
     test:
     - chain: cucushift-installer-check-cluster-health
     workflow: cucushift-installer-rehearse-azure-ipi-private
-- as: azure-ipi-private-network-type-m-accelarated-w-basic-f28
-  cron: 13 13 17 * *
+- as: azure-ipi-priv-network-m-accelarated-w-basic-min-perm-f28
```

Comment:
```suggestion
- as: azure-ipi-priv-network-m-accelarated-w-basic-mini-perm-f28
```

Reply:
And I am thinking if we should move this job to regular ci for more covreage.

Reply:
sure, maybe it can be added into existing ci job, for example, `azure-ipi-vmgenv1-f28`. 

If moving to regular ci, do you have any suggestion about the job name? since we already have job with key words 'minimal-permission', and permission testing in this PR is not minimal ones, but with specific system role.

Reply:
How about `azure-ipi-vmgenv1-specific-permissions-f28`?

Reply:
> maybe it can be added into existing ci job

I suggest to add a new job, in a short future, we are asked to trigger more jobs to get enough data for sippy, so I think adding a new job should be okay.

> If moving to regular ci, do you have any suggestion about the job name

How about `azure-ipi-private-mixed-net-type-mixed-perm-f28`

Reply:
thanks for the suggestion, move job to regular CI and renamed as "azure-ipi-private-spec-net-type-spec-perm-f28"